contributor_path.rst

```diff
@@ -0,0 +1,212 @@
+****************
+Contributor path
+****************
+
+..
+
+  "There is no favorable wind for the sailor who doesn’t know where to go."
+
+  -- Seneca
+
+This guide describes a contributor path in scope of the Ansible project from the early beginning, through all the stages, to becoming a leader shaping the project's future and the future of automation in the IT world in general.
+
+This path is not a policy but rather a suite of recommendations based on experience of successful contributors.
+
+The goal of the path, first of all, is to make your activity more determined and efficient through understanding of what you can achieve and how. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. When you contribute regularly, your proficiency and judgement in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+Regular contribution makes you a member of the community and leads to respect of other community members.
+
+Be persistent and act gradually. Great result is just a product of time and effort. Here's how to get started.
+
+.. note::
+
+   If you have any questions, feel free to reach out to the community in the ``#ansible-community`` `Libera.Chat <https://libera.chat/>`_ IRC channel. If you have ideas on how this document can be improved, please, share them through creating an issue or a pull request.
+
+.. contents::
+
+What is a contribution to an Open Source project
+================================================
+
+First, we should understand what a contribution to an Open Source project is.
+
+The contribution to an Open Source project is any positive addition or change given by an individual or a company to the project including increasing its functionality, improvements to the codebase, code developer experience, user experience, increasing the project's popularity and community, providing infrastructure, and any other kinds of active social or material support.
+
+Note that, according to the classification given in the `Contribution to an Open Source project guide <contribution_to_project.rst>`_, most types of contribution do not require programming skills.
+
+Determine areas of your interests
+=================================
+
+First, determine areas that are interesting to you taking into consideration your current experience and experience you'd like to gain.
+There are some ideas about how you can contribute to the project listed in the `Contribution guide <contribution_to_project.rst>`.
+
+Determine the project
+=====================
+
+The Ansible ecosystem consists of the following projects:
```

Comment:
How is the ``Ansible ecosystem`` defined ? It feels worded in a way that potentially excludes projects that are otherwise listed here: https://docs.ansible.com/ansible/latest/community/other_tools_and_programs.html

If we don't mean to include an exhaustive list of projects here, we should say something along the lines of ``These are some of the Ansible community projects in the ecosystem you could contribute to:``

Reply:
Yes, definitely, there are more projects than listed.. And when i was writing the list I was thinking of some obvious. I'll change it, thanks!